Header header
int64 message_id
int8 class_id
string class_name
float64 goal_x
float64 goal_y
float64 goal_z
float64 distance_from_robot_x
float64 distance_from_robot_y
float64 distance_from_robot_z
